.github/workflows/interop-tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Interoperability tests with GnuTLS and NSS
+on: [pull_request, push]
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 90
+    strategy:
+      fail-fast: false
+      matrix:
+        COMPONENT: [openssl-upstream-gnutls, openssl-upstream-nss]
+    env:
+      COMPONENT: ${{ matrix.COMPONENT }}
+    # Code below inspired by https://fedoramagazine.org/github-actions-use-podman-to-run-fedora-linux/
+    steps:
+      - name: Setup Docker
+        # Docker installation intructions https://docs.docker.com/engine/install/ubuntu/
+        run: |
+          sudo apt-get -y install ca-certificates curl gnupg
+          sudo install -m 0755 -d /etc/apt/keyrings
+          curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+          sudo chmod a+r /etc/apt/keyrings/docker.gpg
+          echo "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+          "$(. /etc/os-release && echo "$VERSION_CODENAME")" stable" | \
+          sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+          sudo apt-get -y update
+          sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+          echo "Docker version"
+          docker version
+      - name : Run tests
+        run: |
+          docker run --sysctl net.ipv6.conf.lo.disable_ipv6=0 -di --name fedora38 ghcr.io/not4pedro/fedora-openssl-interop:fedora38
```

Comment:
`ghcr.io/not4pedro/fedora-openssl-interop`
We have absolutely no clue what is inside this image.

Reply:
Agreed, I've got this PR:
https://github.com/openssl/openssl/pull/22726
In which I'm replacing that container with one we control the building of, and placing it in our github container registry